### PR TITLE
New version: MasterPackager.MasterPackager version 22.1.8131.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56994)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
From the logs:
```
2022-04-06 16:22:08.907 [CLI ] Package hash verification failed. SHA256 in manifest [234df55a28f0b9308fa069228214f1bea831bec8cdf83eca7cf0af2df6722f19] does not match download [8b48c7d1f2305a11de3f813791077bb2bb7c4f15d05e8bce2db4ab92a4c76185]
2022-04-06 16:22:08.917 [CLI ] Terminating context: 0x8a150011 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:17f
```

**Comment by vedantmgoyal2009** [comment]
It didn't put hash validation failed label? (@denelon)

**Comment by OfficialEsco** [comment]
> It didn't put hash validation failed label? (@denelon)

Yeah this seems to be a internal bug which i noticed earlier with a bunch of your PRs which was labeled with `Validation-Installation-Error`

-----

Aaaaand meanwhile typing this comment it got `Error-Hash-Mismatch`

```
4/7/2022 2:06:24 PM
Status: Completed
##[error] Installer https://www.masterpackager.com/installer/public/standard/masterpackager_22.1.8131.0.msi
##[error] Result: Failed
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   234DF55A28F0B9308FA069228214F1BEA831BEC8CDF83ECA7CF0AF2DF6722F19
Expected hash: 8b48c7d1f2305a11de3f813791077bb2bb7c4f15d05e8bce2db4ab92a4c76185
```

Ref org comment:
Manifest: 234df55a28f0b9308fa069228214f1bea831bec8cdf83eca7cf0af2df6722f19
Download: 8b48c7d1f2305a11de3f813791077bb2bb7c4f15d05e8bce2db4ab92a4c76185

So it appears to be correct in this case... interesting

**Comment by ItzLevvie** [comment]
Oops, @denelon please reassign @ryfu-msft

File: manifests/m/MasterPackager/MasterPackager/22.1.8131.0/MasterPackager.MasterPackager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MasterPackager.MasterPackager
+PackageVersion: 22.1.8131.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.masterpackager.com/installer/public/standard/masterpackager_22.1.8131.0.msi
```
Comment (ItzLevvie) [inline manifests/m/MasterPackager/MasterPackager/22.1.8131.0/MasterPackager.MasterPackager.installer.yaml:11]:
```suggestion
  InstallerUrl: https://www.masterpackager.com/installer/masterpackager_22.1.8131.0.msi
```
Have you tried this?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**